### ISSUE TITLE
Hide isponsorBlock

### ISSUE THREAD

**Issue by dinhthengu** [arichornlover/uYouEnhanced#792]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Do you think this is a bug?

✅ Yes, I believe this is a bug. I will open a new issue with the bug template

### My question

![IMG_5422](https://github.com/user-attachments/assets/5579d656-7d41-41ba-a6b8-82d946dba1f8)

Hi everyone. please help me how to hide the isponsorBlock button, i have enabled hide in enhanced but it doesn't work. the version i am using is the latest. thanks all

**Comment by dinhthengu** [comment]
> Rebuild the IPA. It’s been fixed recently.

Many thanks bro. 
I hope the next update will fix it. Because i don't know how to build ipa 😭😭

**Comment by JudeTheTechSavvyDude** [comment]
when is the actual releases gonna be updated for the latest version?

**Comment by dinhthengu** [comment]
> > when is the releases gonna be updated for the latest version?
> 
> It’s been updated today! uYouEnhanced 19.42.1-3.0.4!

Yes bro, I have just updated and it works great 🙂Many many thank you !! 👍👍👍👍👍👍
